src/types/kongponents-plugin.ts

```diff
@@ -0,0 +1,12 @@
+import { v4 as uuidv4 } from 'uuid'
+
+export const kongponentsRandomIdFunction = 'kongponents-id' as const
+
+export interface KongponentsPlugin {
+  kongponentsId: () => string
+}
+
+export const defaultKongponentsPlugin: KongponentsPlugin = {
+  kongponentsId: uuidv4,
```

Comment:
This will impact other files, but since this is an option within the `Kongponents` plugin, we don't need to prefix it
```suggestion
  uniqueIdFn: () => string
}

export const defaultKongponentsPlugin: KongponentsPlugin = {
  uniqueIdFn: uuidv4,
```